test/index_tests.js

```diff
@@ -317,6 +319,42 @@ it('WHEN mongoose models are supplied THEN the definitions should be included',
   });
 });
 
+it('WHEN tags are supplied THEN the paths should be contain the matching ones', done => {
```

Comment:
if we start using mongoose model names that test could be merged with previous one just by adding extra assertion there.

Reply:
Could not reply to this one: https://github.com/mpashkovskiy/express-oas-generator/pull/62#discussion_r464862638.

For those using mongooseModels probably both arguments would be the same (as in `server_basic.js` and `server_advanced.js` examples).
But in my opinion they are **independent features**. 
For someone using sequalize, it wouldn't provide the definitions spec but could be grouping by tags.

Reply:
I can merge that tests anyway to simplify. The coverage would be exactly the same.

Reply:
Good point, indeed tags could be used independently. But how about by default do grouping on mongoose models even if no tags are provided?

Reply:
Done!